open_humans/templates/base.html

```diff
@@ -94,7 +94,7 @@
                   aria-haspopup="true" aria-expanded="false">My account
                   <span class="caret"></span></a>
               <ul class="dropdown-menu">
-                <li><a href="{% url 'my-member-dashboard' %}">Public profile</a></li>
+                <li><a href="{% url 'member-detail' member.user.username %}">Public profile</a></li>
```

Comment:
Can we change this to 'member-detail-direct' instead?

'member-detail' is nice to retain as an unambiguous fallback, but I think people like having a profile URL that looks like openhumans.org/username :)